### DISCUSSION TITLE
Bug: Pull Requests page shows PRs from random people not just my PRs

### DISCUSSION THREAD

**Discussion by ninalouw** [community/community#38017] [Pull Requests] [answered]
Click on Pull Requests
See that the page https://github.com/pulls displays Pull Requests from many random people, not just the user's pull requests
![Firefox_Screenshot_2022-11-01T19-58-44 966Z](https://user-images.githubusercontent.com/17684394/199327010-679949fe-f91e-4066-8d55-d1d8176e74b3.png)

**Comment by david-engelmann** [comment, marked as answer]
This bug has been reported in a thread below. It seems that you can sill get your info by pressing enter after entering the search bar

**Comment by macMikey** [comment]
is there any update on this problem? i created a commit to one of my repos, and then PR'd it to the parent repo. in both my repo and in the parent repo, a different account is listed as the author.
i have no idea who mustache-inc is, but it is not me.
here is the commit in my repo
https://github.com/macMikey/4D-NetKit/commit/61c5d288e3e419fa1db7ea516e6f30ec5fb3b4c6
here is the pr:
https://github.com/4d/4D-NetKit/pull/60
